x11-terms/zutty/zutty-0.8.ebuild

```diff
@@ -16,15 +16,15 @@ LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
 
-COMMON_DEPEND="
+RDEPEND="
 	media-libs/freetype:2
 	media-libs/libglvnd[X]
-	x11-libs/libXmu"
-DEPEND="
-	${COMMON_DEPEND}
-	${PYTHON_DEPS}"
-RDEPEND="${COMMON_DEPEND}"
-BDEPEND="virtual/pkgconfig"
+	x11-libs/libXmu
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+"
```

Comment:
Since this is waf-based, aren't the contents of *RDEPEND* needed in *DEPEND* too? Just let me know, I can add it later.